secant/Features/SendFlow/SendFlowStore.swift

```diff
@@ -276,22 +279,38 @@ extension SendFlowViewStore {
 
     var bindingForConfirmation: Binding<Bool> {
         self.routeBinding.map(
-            extract: { $0 == .confirmation || self.bindingForSuccess.wrappedValue || self.bindingForFailure.wrappedValue },
+            extract: {
+                $0 == .confirmation ||
```

Comment:
This seems Pretty complicated and hard to test. If navigation depends on it I can foresee bugs from changes in other parts of the code.

Reply:
This is how currently routing works in the Secant app. This is first case when we want to route multiple screens in chain so the code looks like this. I agree that it's not ideal. I suggest to not do anything with this in this PR and address routing as another issue.

Reply:
Great, let's create a new issue that defines the follow up work. I consider this resolved then